API-Server/src/main/java/swm/hkcc/LGTM/app/modules/mission/repository/MissionCustomRepositoryImpl.java

```diff
@@ -47,6 +53,17 @@ private List<Mission> getMissions(BooleanExpression isParticipating, BooleanExpr
                 .join(missionRegistration).on(mission.missionId.eq(missionRegistration.mission.missionId))
```

Comment:
이건 질문인데,
이렇게 join문을 작성해도 잘 동작하더라구요!
id를 참조해서 조회하는 것이 보다 이점이 있나요?
```suggestion
                .join(missionRegistration).on(mission.eq(missionRegistration.mission))
```

Reply:
아마 실제 실행되는 SQL은 동일할 것 같습니다

Reply:
그렇군요!